### PR TITLE
rpm: execute install/remove/upgrade test

### PR DESCRIPTION
**Pull request by kenhys**
Closes: #167

### CODE REVIEW

**Comment by kenhys** [comment]
@ashie Could you review this PR, please?

File: td-agent/yum/install-test.sh [hunk 1]
```diff
@@ -14,11 +14,13 @@ set -exu
 distribution=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f2)
 version=$(cat /etc/system-release-cpe | awk '{print substr($0, index($1, "o"))}' | cut -d: -f4)
 
+ENABLE_UPGRADE_TEST=1
 case ${distribution} in
   amazon)
     case ${version} in
       2)
         DNF=yum
+	ENABLE_UPGRADE_TEST=0
```
Comment (ashie) [inline td-agent/yum/install-test.sh:23]: indent

File: td-agent/yum/install-test.sh [hunk 2]
```diff
@@ -29,13 +31,48 @@ case ${distribution} in
         ;;
       *)
         DNF="dnf --enablerepo=PowerTools"
+	ENABLE_UPGRADE_TEST=0
```
Comment (ashie) [inline td-agent/yum/install-test.sh:34]: indent

File: td-agent/yum/install-test.sh (hunk 1 above)
Comment (ashie) [reply td-agent/yum/install-test.sh:23]: Is it O.K. to skip upgrading test for Amazon Linux 2? Following code seems trying to do it for this platform...

File: td-agent/yum/install-test.sh (hunk 2 above)
Comment (kenhys) [reply td-agent/yum/install-test.sh]: @ashie Thanks, I've fixed it.

File: td-agent/yum/install-test.sh (hunk 1 above)
Comment (kenhys) [reply td-agent/yum/install-test.sh]: @ashie Thanks, I've fixed wrong baseurl, so I've enabled test again.

**Pull request merged**